### ISSUE TITLE
Blocking metrics updates on test regressions

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/interop#276]
As discussed in https://github.com/web-platform-tests/results-analysis/issues/143, any new non-OK harness status (for testharness.js tests) will prevent the Interop 2023 scores from updating until that test is added to the [list of ignored tests statuses](https://github.com/web-platform-tests/results-analysis/blob/12bd4ed4c607eb3bc6a340637c9fe5f55169f45a/interop-scoring/main.js#L36-L39).

This mechanism was inherited from Interop 2022. During 2022, there were no regressions, so we never noticed this.

Unfortunately, https://github.com/web-platform-tests/wpt/pull/38258 broke the score updating for almost a week before @gsnedders alerted me to the fact in https://github.com/web-platform-tests/results-analysis/issues/143 and I just fixed it.

I don't think we have the right tradeoff here, we shouldn't block Interop 2023 updates because a regression in a single test, and we shouldn't require the interop team to triage and fix such regressions.

I think we should:
- Land the change in https://github.com/web-platform-tests/results-analysis/pull/142 and do the same for Interop 2023 scoring
- In the scoring script, log changes to test scores to a text file, one per focus area. That can be used both to identify both improvements to celebrate and regressions to investigate.
- Investigate fixing https://github.com/web-platform-tests/wpt.fyi/issues/3149 so that we can query for all harness errors in the latest test results.

**Comment by gsnedders** [comment]
The thing that's obvious here is how neither https://github.com/web-platform-tests/wpt/pull/38203 nor https://github.com/web-platform-tests/wpt/pull/38258 failed CI on WPT (see https://github.com/web-platform-tests/wpt/issues/10877).

**Comment by foolip** [comment]
@gsnedders I didn't confirm it, but assumed based on the timing (~4 hours apart) that both PRs passed the CI separately and were then both merged.

**Comment by foolip** [comment]
This is blocked on getting review of https://github.com/web-platform-tests/results-analysis/pull/150.

**Comment by foolip** [comment]
Closing this as I won't be driving any changes here. I've closed https://github.com/web-platform-tests/results-analysis/pull/150 but note that https://github.com/web-platform-tests/results-analysis/pull/173 remains open. @jgraham can you file a new issue if you want to move forward with some change?
